x-pack/plugins/watcher/public/sections/watch_edit/components/json_watch_edit/json_watch_edit_form.tsx

```diff
@@ -59,21 +59,21 @@ export const JsonWatchEditForm = () => {
 
   return (
     <Fragment>
-      {validationResult && validationResult.type === 'warning' && (
-        <ConfirmWatchesModal
```

Comment:
What was this confirmation modal previously used for?

Reply:
There previously was a modal that rendered when a user attempts to create a watch with the same id of an existing watch. I looked at our other apps, and it looks like the pattern we use is to render an error in this scenario (when there is a conflict), so I went with this approach.